disnake/app_commands.py

```diff
@@ -661,6 +685,7 @@ def from_dict(cls, data: ApplicationCommandPayload) -> Self:
             name=Localized(data["name"], data=data.get("name_localizations")),
             dm_permission=data.get("dm_permission") is not False,
             default_member_permissions=_get_as_snowflake(data, "default_member_permissions"),
+            nsfw=data.get("nsfw"),
```

Comment:
should this default to False here, instead of None?

Reply:
It'll default to False in the constructor:
https://github.com/DisnakeDev/disnake/blob/54cbd12cdf8da764d4032ad31a13b3665821cd16/disnake/app_commands.py#L500